apple/Resources/FluentUI-apple.xcassets/FluentColors/gray900.colorset/Contents.json

```diff
@@ -1,20 +1,20 @@
 {
-  "info" : {
-    "version" : 1,
-    "author" : "xcode"
-  },
   "colors" : [
     {
-      "idiom" : "universal",
       "color" : {
         "color-space" : "srgb",
         "components" : {
-          "red" : "0x21",
           "alpha" : "1.000",
           "blue" : "0x21",
-          "green" : "0x21"
+          "green" : "0x21",
```

Comment:
why did this file change?

Reply:
oh oops, i must've clicked something on accident when checking the hex. i'll revert